### PR TITLE
Menu.php begin and very minor edits to Reserve.php

### PR DESCRIPTION
**Pull request by chasestubb**
I'm sure this needs work. Just wanted to get some feedback. Thanks!

### CODE REVIEW

File: menu.php [hunk 1]
```diff
@@ -11,11 +12,30 @@
   <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
 </head>
 
+<?php
+// PHP HEADER
+	include "header.php";
+
+	// change the value of $dbuser and $dbpass to your username and password
+	include 'connectvars.php';
```
Comment (Bensonre) [inline menu.php:20]: use config not connect vars

Comment (Bensonre) [reply menu.php:20]: The connect already occurs in the config file as well so you wont need to cal the connection

**Review by Bensonre — DISMISSED** [review]
This didn't even load, i got a 500 hundred error when I checked it out. I will finish this review later

File: menu.php (hunk 1 above)
Comment (chasestubb) [reply menu.php:20]: testing revealed this was not the problem. figured it out though!

**Review by chasestubb — COMMENTED** [review]
Deployment Script is all good. There was initially a merge conflict (pictures table), but it was simple to resolve.

**Pull request merged**